emcc.py

```diff
@@ -2663,6 +2663,9 @@ def get_full_import_name(name):
 
     if settings.ASYNCIFY == 2:
       diagnostics.warning('experimental', 'ASYNCIFY with stack switching is experimental')
+    else:
+      settings.REQUIRED_EXPORTS += ['emscripten_fiber_init',
+                                    'emscripten_fiber_init_from_current_context']
```

Comment:
Why do we need to export these two functions?  I don't see any usage of them in JS.. but maybe I'm missing something?

Reply:
Previously I was getting link errors that suggested I should export them, despite them only being used in C code. That was when they still were in a separate library. I've just confirmed that it's not actually needed, at least not anymore. It probably wasn't needed back then either and I just got confused.